### PR TITLE
[READY] Blog about self directing teams

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
This is a blog from Marian about self directing teams and it is ready for review. 

Would like to publish this next thursday morning.

### CODE REVIEW

File: source/blog/2016-04-14-zelfsturende-teams-hot-topic-in-de-zorg.html.markdown [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+## Waarom kiezen organisaties eigenlijk voor zelfsturende teams?
+Organisaties hebben de ambitie om de dienstverlening continu te verbeteren vanuit het cliënt centraal perspectief. Om dat te realiseren is het van belang de denk- en doe-kracht van iedere zorgmedewerker optimaal te gebruiken. Medewerkers hebben binnen zelfsturende teams meer vrijheid en eigen beslissingsbevoegdheid. De ervaring leert dat er daardoor minder verzuim en minder stress is.
+
+## Wat zijn belangrijke succesfactoren?
+Zelfsturing betekent niet dat er geen sturing is. Uit ervaring blijkt dat zelfsturing niet kan zonder harde kaders en het bewustzijn van medewerkers wat hun cirkel van invloed is. Top down is er real time sturingsinformatie nodig om zelfsturing optimaal te faciliteren en te ondersteunen. Een succesvolle implementatie van zelfsturende teams begint bij een context waarin er transparantie en inzicht is in de gestelde kaders. De uitdaging voor organisaties is het ontzorgen van de teams om hun core business uit te voeren.
```
Comment (fatboypunk) [inline source/blog/2016-04-14-zelfsturende-teams-hot-topic-in-de-zorg.html.markdown]: The `sturingsinfromatie` is bottom up here right? the information comes from the team to management so they can set the guidelines

File: source/blog/2016-04-14-zelfsturende-teams-hot-topic-in-de-zorg.html.markdown [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+## Wat draagt het Kwaliteitspaspoort van Defacto hier aan bij?
+Defacto heeft met een aantal zorginstellingen het [CAPP Kwaliteitspaspoort](/kwaliteitspaspoort) ontwikkeld die de benodigde real time data genereert op het gebied van vakbekwaam- en -bevoegdheid. Het inzetten van het Kwaliteitspaspoort geeft de organisatie, het team en de individu de noodzakelijke sturingsinformatie. Door de gezamenlijke verantwoordelijkheden en de overlap van taken kunnen teamleden functievervaging ervaren. Waar ze voorheen een duidelijke verantwoordelijkheid hadden, krijgen ze nu meer te maken met operationele taken en randzaken die ook binnen het zelfsturende team uitgevoerd moeten worden. Door inzicht in vakbekwaam- en bevoegdheden en competenties in het Kwaliteitspaspoort kan het team er zelf in sturen.
```
Comment (fatboypunk) [inline source/blog/2016-04-14-zelfsturende-teams-hot-topic-in-de-zorg.html.markdown]: `Door inzicht in vakbekwaam- en bevoegdheden en competenties` -> `Door inzicht in competenties, vakbekwaam- en bevoegdheden`

**Comment by fatboypunk** [comment]
Personally i think it would make more sense to say information needs to flow bottom up, so management can set guidelines for the teams. Might be me but from this post I don't get how it's going to help team taking control

**Comment by RonaldDijkstra** [comment]
I think you make a valid point. I will contact Marian about this. Thanks Marcel.

Op 12 apr. 2016 om 18:36 heeft Marcel Horlings <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:


Personally i think it would make more sense to say information needs to flow bottom up, so management can set guidelines for the teams. Might be me but from this post I don't get how it's going to help team taking control

-

**Comment by RonaldDijkstra** [comment]
I think removing top down makes sense!

> Zelfsturing betekent niet dat er geen sturing is. Uit ervaring blijkt dat zelfsturing niet kan zonder harde kaders en het bewustzijn van medewerkers wat hun cirkel van invloed is. Daarom is real time sturingsinformatie ...

File: source/blog/2016-04-14-zelfsturende-teams-hot-topic-in-de-zorg.html.markdown [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+## Wat draagt het Kwaliteitspaspoort van Defacto hier aan bij?
+Defacto heeft met een aantal zorginstellingen het [CAPP Kwaliteitspaspoort](/kwaliteitspaspoort) ontwikkeld die de benodigde real time data genereert op het gebied van vakbekwaam- en -bevoegdheid. Het inzetten van het Kwaliteitspaspoort geeft de organisatie, het team en de individu de noodzakelijke sturingsinformatie. Door de gezamenlijke verantwoordelijkheden en de overlap van taken kunnen teamleden functievervaging ervaren. Waar ze voorheen een duidelijke verantwoordelijkheid hadden, krijgen ze nu meer te maken met operationele taken en randzaken die ook binnen het zelfsturende team uitgevoerd moeten worden. Door inzicht in competenties, vakbekwaam- en bevoegdheden in het Kwaliteitspaspoort kan het team er zelf in sturen.
```
Comment (fatboypunk) [inline source/blog/2016-04-14-zelfsturende-teams-hot-topic-in-de-zorg.html.markdown]: maybe an enter after `noodzakelijke sturingsinformatie` and before `Door`

Comment (RonaldDijkstra) [inline source/blog/2016-04-14-zelfsturende-teams-hot-topic-in-de-zorg.html.markdown]: Done, makes it more readable.

**Comment by RonaldDijkstra** [comment]
I asked the author and she is fine with the changes. So this blog is ready-ready. :+1:

**Pull request merged**